### DISCUSSION TITLE
Issue handling

### DISCUSSION THREAD

**Discussion by leamas** [opencpn-manuals/main#8] [General]
Perhaps we need to talk about issues. From https://github.com/rgleason/ocpn-dev-manual/issues/82#issuecomment-896864588https://github.com/rgleason/ocpn-dev-manual/issues/82#issuecomment-896864588: 

> This isnt a bug. This is a reminder for how to work, indeed it is your instructions!

Should we try to keep to the general idea that a bug is an "actionable item" also here? And keep other things including like these in this discussion forum?

**Comment by Rasbats** [comment]
Fine at this end.

**Comment by rgleason** [comment]
Well, we have "BUGS" and "RFE"  why not use Issues in this context as a "REM" reminder.
If it is a reminder to do something to the documents, that is actionable....
If it is a reminder to add a topic to the documents, that is actionable...
If it is a reminder to consider some aspect, that is actionable.
If it is a reminder for a sequence of commands that are suggested for some part of the process, that should become part of the documentation.

The "Issues" in my ocpen-dev-manual were often used as a way of tagging something important that should not be lost in closed Issues.

**Reply by leamas** [reply]
I don't see the use for such a tag. All things you mention should basically be a bug or an RFE depending on the situation. All bugs are per definition "reminders" as  I see it.
